modules/dr-restoring-cluster-state.adoc

```diff
@@ -290,20 +290,15 @@ etcd-ip-10-0-143-125.ec2.internal                1/1     Running     1
 ----
 +
 If the status is `Pending`, or the output lists more than one running etcd pod, wait a few minutes and check again.
-+
-[NOTE]
-====
-Perform the following step only if you are using `OVNKubernetes` network plugin.
-====
 
-. Delete the node objects that are associated with control plane hosts that are not the recovery control plane host.
+. If you are using the `OVNKubernetes` network plugin, delete the node objects that are associated with control plane hosts that are not the recovery control plane host.
```

Comment:
Nice, this is a much better way to document the conditional step compared to having that note above.